### PR TITLE
52 price handle

### LINKED ISSUE #52: BUG: cannot handle decimal as price input for find and find_all
Spec harness issue example:
Issue example #1:
ItemsApiTest#test_loads_individual_items [/Users/Laszlo/turing/3module/rales_engine_spec_harness/test/endpoints/items_test.rb:22]:
Expected: "774.84"
  Actual: 77484

Issue example #2:
InvoiceItemsApiTest#test_it_can_find_first_instance_by_unit_price:
NoMethodError: undefined method `[]' for nil:NilClass  /Users/Laszlo/turing/3module/rales_engine_spec_harness/test/endpoints/invoice_items_test.rb:81:in `test_it_can_find_first_instance_by_unit_price'

Issue example #3:
InvoiceItemsApiTest#test_loads_individual_invoice_items [/Users/Laszlo/turing/3module/rales_engine_spec_harness/test/endpoints/invoice_items_test.rb:17]:
Expected: "219.16"
  Actual: 21916

### PR DESCRIPTION
**Pull request by Laszlo-JFLMTCO**
@akintner This PR closes #52 . Now we are properly formatting `unit_price` both ways: when we output it through the JSON response as well as when we receive it as input through the API call.

Corresponding `rake` tests are passing, too.

FYI @case-eee

### CODE REVIEW

**Comment by akintner** [comment]
nice, looks solid. Merging now

File: app/serializers/item_serializer.rb [hunk 1]
```diff
@@ -1,3 +1,9 @@
 class ItemSerializer < ActiveModel::Serializer
+  include ModelHelpers
+
   attributes :id, :name, :description, :unit_price, :merchant_id, :created_at, :updated_at
+
+  def unit_price
```
Comment (case-eee) [inline app/serializers/item_serializer.rb:6]: i think you could almost define your penny_to_dollar method here. what's the purpose of this method? who needs access to it?

File: app/helpers/model_helpers.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+module ModelHelpers
```
Comment (case-eee) [inline app/helpers/model_helpers.rb:1]: your helper modules are intended for your views - can we find another place to put these methods?

**Review by case-eee — COMMENTED** [review]
@Laszlo-JFLMTCO @akintner i suggest finding another place for those penny helpers rather than the helper module. i think overall, this does accomplish your goal. so keep that in mind. maybe this is an opportunity to refactor later!

File: app/serializers/item_serializer.rb (hunk 1 above)
Comment (Laszlo-JFLMTCO) [reply app/serializers/item_serializer.rb:6]: @case-eee Thank you for the comment. The thinking behind extracting it into a `helper method` was that we use `unit_price` in multiple categories so we have a single module responsible for the conversion methods.

FYI @akintner

File: app/helpers/model_helpers.rb (hunk 2 above)
Comment (Laszlo-JFLMTCO) [reply app/helpers/model_helpers.rb:1]: @case-eee Thank you for the comment! I must have misunderstood the purpose of the helper. It wasn't very clear that it is for `views` only. Do you have another suggestion?
@akintner

Comment (case-eee) [reply app/helpers/model_helpers.rb:1]: @Laszlo-JFLMTCO @akintner i think you could create a module or a PORO in your models directory. agreed - it's not super clear that they are intended for your views. it's pretty subtle - but they are automatically available in every view template you write, but you have to manually include them everywhere else you want to use them

Comment (Laszlo-JFLMTCO) [reply app/helpers/model_helpers.rb:1]: Thank you @case-eee This explanation helps! It was weird why I had to `include` it manually at multiple places...
FYi @akintner

**Pull request merged**